src/transformers/modeling_flax_utils.py

```diff
@@ -660,6 +665,14 @@ def from_pretrained(
                     )
             elif os.path.isfile(pretrained_model_name_or_path) or is_remote_url(pretrained_model_name_or_path):
                 archive_file = pretrained_model_name_or_path
+            # check if an index file exists
+            elif hf_url_exists(pretrained_model_name_or_path, WEIGHTS_INDEX_NAME):
```

Comment:
This will make one remote call to the API for all models, which will slightly slow down all calls to `from_pretrained`. I'd rather have this in an except later on like for PyTorch models. Also isn't there a `from_pt` missing here?

Reply:
let me check that! 
Also totally agree, will design an except for that 👍🏻